### PR TITLE
Comment on PR if validation fails

### PR DESCRIPTION
**Pull request by prasunanand**
Fixes https://github.com/Quansight/qhub/issues/7

### CODE REVIEW

**Comment by prasunanand** [comment]
Example: https://github.com/Quansight/qhub-gcp-test-deployment/pull/1#issuecomment-707298719

File: qhub/template/{{ cookiecutter.repo_directory }}/.github/workflows/qhub-validate.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  qhub-validate:
+    name: 'Qhub'
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout Infastructure'
+        uses: actions/checkout@master
+      - name: 'Qhub Format'
+        uses: Quansight/qhub-github-actions@v0.1-beta
+        with:
+          qhub_actions_version: 0.0.1
```
Comment (costrouc) [inline qhub/template/{{ cookiecutter.repo_directory }}/.github/workflows/qhub-validate.yaml:25]: Does this allow us to use the qhub version 0.1.19 for validation?

Comment (prasunanand) [reply qhub/template/{{ cookiecutter.repo_directory }}/.github/workflows/qhub-validate.yaml:25]: The version field doesn't do anything right now. I have kept it for a later stage when the qhub-bot can do other tasks as well. Should I remove version field ?

**Comment by laisbsc** [comment]
@costrouc is this linked to #7? Do you know what is the status on it?

**Comment by tylerpotts** [comment]
Superceded by #408. Closing

**Pull request closed without merging**